### PR TITLE
25 shiga

### PR DESCRIPTION
**Pull request by shomiyazaki**
New SMC.

Hi @kentoyamada100 The SMC run with the method that Shiro suggested in Slack. It would be appreciated if you start reviewing them. I am looking forward to your feedback and comments!

### CODE REVIEW

File: analyses/25_shiga/02_sim_25_shiga.R [hunk 1]
```diff
@@ -92,9 +92,11 @@ run_simulations <- function(pref_n, prefadj_n){
                                    adj = prefadj_n)
 
   # Run simulation
+  set.seed(2020)
```
Comment (kentoyamada100) [inline analyses/25_shiga/02_sim_25_shiga.R:95]: Screen Shot 2022-06-12 at 15 34 54
Screen Shot 2022-06-12 at 15 35 07
Is this the latest version of the code? I still get this error... Were you able to resolve this issue?

**Review by kentoyamada100 — CHANGES_REQUESTED** [review]

**Comment by shomiyazaki** [comment]
@kentoyamada100 Hi! Shiga is ready with new `runs=4L`. Thank you in advance, and I am looking forward to your reviews and comments

File: analyses/25_shiga/02_sim_25_shiga.R [hunk 2]
```diff
@@ -135,7 +138,7 @@ run_simulations <- function(pref_n, prefadj_n){
                                 "_",
                                 as.character(sim_type),
                                 "_",
-                                as.character(nsims),
+                                as.character(nsims * 4),
                                 "_",
```
Comment (kentoyamada100) [inline analyses/25_shiga/02_sim_25_shiga.R:142]: Please delete the old data.
Screen Shot 2022-06-18 at 18 33 30

File: analyses/25_shiga/25_shiga_hr_2020_doc.md [hunk 3]
```diff
@@ -45,7 +45,7 @@
 * [衆議院議員選挙の小選挙区のポリゴンデータ (東京大学空間情報科学研究センター 西沢明）](https://home.csis.u-tokyo.ac.jp/~nishizawa/senkyoku/)
 
 ## データの処理
-特になし
+`02_sim_25_shiga.R`にてシュミレーション（`redist_smc()`）を執行する際に、乱数のシードをランダムに設定した。これは、4つの独立したSMC (`runs = 4L`) を執行する際にランダムに発生するプログラムのエラーに対応するためである。なお、1つ (`1L`)と2つ(`2L`)の独立したSMCによって発生する結果や分布と比較することで、チェリーピッキングを回避した。
```
Comment (kentoyamada100) [inline analyses/25_shiga/25_shiga_hr_2020_doc.md:48]: You should put this in the section above this one ("設定"). "設定" is where we talk about the method we used. "データの処理" is where we explain the pre-processing steps (which we mainly do in 01.R)

**Review by kentoyamada100 — CHANGES_REQUESTED** [review]
Please make the suggested changes and please also make sure to upload the new data to Dataverse, if you haven't already

**Pull request merged**